opensearchapi/api_search.go

```diff
@@ -91,6 +91,7 @@ type SearchHit struct {
 	ID          string                  `json:"_id"`
 	Score       float32                 `json:"_score"`
 	Source      json.RawMessage         `json:"_source"`
+	Fields      json.RawMessage         `json:"fields"`
```

Comment:
I think this new field `Fields` should also be added to https://github.com/opensearch-project/opensearch-go/blob/main/opensearchapi/api_search.go#L65.

Reply:
Can you show the example of request that will response with `fields` in `SearchResp` ?

Reply:
Actually its my bad! You have already added it in the `SearchHit` struct which is correct.